website/docs/r/firewall_shared.html.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: "ibm"
```

Comment:
change firewall-shared to hardware-firewall-shared as we have changed naming in all required places